src/components/calcite-flow/calcite-flow.tsx

```diff
@@ -21,6 +21,11 @@ export class CalciteFlow {
    */
   @Prop({ reflect: true }) theme: CalciteTheme;
 
+  /**
+   * When true, Flow responds to the back event of flow item
+   */
+  @Prop({ reflect: true }) handleBack = false;
```

Comment:
The flow seemed to be missing an implementation that didn't require additional boilerplate to wire it up. This new config option provides a simple means of seeing the flow in action. `false` by default.

Reply:
fixme: I'd prefer using a promise instead of this.

```tsx
@Prop() beforeBack: (el: HTMLElement) => Promise<void> = () =>
```

See: https://github.com/Esri/calcite-components/blob/master/src/components/calcite-modal/readme.md

Reply:
discussed with @driskull and am going to move this to a separate PR, because the modified approach would be a breaking change.